tap_swellrewards/schemas/customers.json

```diff
@@ -104,6 +104,26 @@
       ],
       "format": "date-time"
     },
+    "vip_tier_actions_completed": {
+      "type": [
+        "object",
+        "null"
+      ],
+      "properties": {
+        "points_earned": {
+          "type": "integer"
```

Comment:
We should make these fields nullable

Reply:
Ah yes

Reply:
Although technically if the object is present, these fields will be present as well, but I'll make them nullable anyway.

Reply:
@luandy64 Updated

Reply:
> Although technically if the object is present, these fields will be present as well, but I'll make them nullable anyway.

I had a feeling this was true. 

I should've made my comment a question because I know nothing about this tap/API. I just think it's silly when taps fail because the API doesn't do what it says it'll do - like always return a field. So feel free to make these not nullable, or keep them nullable if you don't really care 👍